src/app/modules/main/chat_section/chat_content/chat_details.nim

```diff
@@ -249,18 +249,18 @@ QtObject:
     self.isContact = value
     self.isMutualContactChanged()
 
-  proc isUntrustworthyChanged(self: ChatDetails) {.signal.}
-  proc getIsUntrustworthy(self: ChatDetails): bool {.slot.} =
-    return self.isUntrustworthy
-  QtProperty[bool] isUntrustworthy:
-    read = getIsUntrustworthy
-    notify = isUntrustworthyChanged
+  proc trustStatusChanged(self: ChatDetails) {.signal.}
+  proc getTrustStatus(self: ChatDetails): int {.slot.} =
```

Comment:
I like this! Hopefully this will allow us to set the green check mark on the client side only; is that what you're after here? :)

Reply:
Pretty much. It started as a simplification of the code. We have `isUntrustworthy` and `isVerified`, but both of them come from `trustStatus`, so it's way simpler for the Nim code to just export `trustStatus` and either in the model expose `isUntrustworthy` or check in QML.

BTW, this is now back to being WIP. I'll let you know when I'm done. I found more places to clean up and also it should fix the trust checkmark issues